active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,225 @@
+Breadcrumb control
+===
+
+# Background
+
+A "Breadcrumb" UX pattern provides an way to show a current position in context.
+For example in Windows File Explorer the current directory is shown with a breadcrumb,
+where clicking on any node takes you to that ancestor directory:
+
+![File Explorer breadcrumb example](images/file-explorer-breadcrumb.jpg)
+
+There’s currently no consistent and Fluent way to address the common UX pattern of a breadcrumb.
+This control is needed for situations where the user's navigation trail (in a file system or menu system) needs
+to be persistently visible.
+
+A Breadcrumb control is movitated by its use in many app scenarios and supporting developers in migrating from WPF.
+This is not the 'history' of the user's navigation, but the direct path of pages or folders from
+the root node to their current position. 
+
+## V1/V2 Breadcrumb
+
+The work for Breadcrumb has been scoped to V1 and V2 stages of the API.
+In V1 the nodes have no option to open a drop-down menu, typically used to show the node's siblings.
+V2 will add this. This spec focuses on V1 Breadcrumb,
+but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# API Pages
+
+## Breadcrumb class
+
+Represents a control that shows a list of items, for example the navigation trail to the current location.
+
+### Examples
+
+The following example code creates a simple Breadcrumb control with three nodes.
+When there is not enough width to display the whole control,
+the leading nodes are replaced with a node that the user can click on,
+opening the missing nodes in a flyout menu.
```

Comment:
Is the flyout "..." shorter by design? I can't stop looking at how the "..." and ">" are not vertically aligned with the other items in the trail.

Reply:
The pictures don't show the final visual design.  @kat-y can you update the pictures ?

Reply:
Recommend: update to latest